csharp/test/Microsoft.ML.OnnxRuntime.EndToEndTests/runtest-docker.sh

```diff
@@ -16,14 +19,25 @@ docker build -t "onnxruntime-$IMAGE" --build-arg OS_VERSION=16.04 --build-arg PY
 
 
 docker rm -f "onnxruntime-cpu" || true
+
+set +e
+
 docker run -h $HOSTNAME \
         --rm \
         --name "onnxruntime-cpu" \
         --volume "$SOURCE_ROOT:/onnxruntime_src" \
         --volume "$BUILD_DIR:/home/onnxruntimedev" \
-        --volume "$HOME/.cache/onnxruntime:/root/.cache/onnxruntime" \
+        --volume "$HOME/.cache/onnxruntime:/home/onnxruntimedev/.cache/onnxruntime" \
         "onnxruntime-$IMAGE" \
         /bin/bash /onnxruntime_src/csharp/test/Microsoft.ML.OnnxRuntime.EndToEndTests/runtest.sh \
-        /home/onnxruntimedev/$NUGET_REPO_DIRNAME /onnxruntime_src &
+        /home/onnxruntimedev/$NUGET_REPO_DIRNAME /onnxruntime_src /home/onnxruntimedev $TestDataUrl $TestDataChecksum &
```

Comment:
Are we setting TestDataUrl outside, or is it another positional argument on the command line?

Reply:
it is set as a build environment variable

Reply:
this script should be usable from command line as well, outside the build. It's the primary script for building on Linux.